.github/workflows/docker-build-backend-container-on-merge-group.yml

```diff
@@ -0,0 +1,36 @@
+name: Build Backend Image on Merge Group
+
+on:
+  pull_request:
+    branches: [ "main" ]
+  merge_group:
+    types: [checks_requested]
+
+env:
+  REGISTRY_IMAGE: danswer/danswer-backend
+
+jobs:
+  build:
+    # for EE, run on special image-builders runners
```

Comment:
There's no longer a separate EE set of actions, let's just remove this comment here